docs/src/dev_guide/inputs.md

```diff
@@ -0,0 +1,337 @@
+
+ - **index_rps.csv**: [Renewable Potfolio Standard (RPS)](https://www.nrel.gov/state-local-tribal/basics-portfolio-standards.html) qualified technologies. Consists of a single row containing qualified technolgies, for e.g., `biopower`, `wind-ofs` etc.
+
+ - **index_stor_existing.csv**: Existing bulk storage technologies. Consists of a single row containing the existing technolgies, for e.g., `battery_1`, `battery_2` etc.
+
+ - **index_stor_new.csv**: Potential bulk storage technologies. Consists of a single row containing the existing technolgies, for e.g., `battery_1`, `battery_2` etc.
+
+ - **index_l.csv**: Transmission lines. Each line is represented as `\$(origin_zone)||\$(destination_zone)`, for e.g., `p100||p101`.
+
+ - **index_p.csv**: Individual independent power producers (IPP). Cosnsists of a single row of IPP names.
+
+#### Customer
+
+ - **index_m.csv**: Behind-The-Meter (BTM) technologies. Eaxample values are `BTMPV, BTMStorage`.
+
+## Parameters
+
+### Model Parameters
+
+ - **Omega.csv**: Number of days per representative day. Column names are the representative days in index_d and the values are the number of days for that representative day.
+
+ - **Gamma.csv**: Number of customers of type in index\_h at zone in index\_z. Column names are index\_z, followed by the customer groups (for e.g., `index_z, Commercial, Residential, Industrial`).
+
+ - **Year.csv**: Column names and values are the representative years in index_y.
+
+ - **Time.csv**: Number of days per representative hour. Column names are the representative hours in index_t and the values are the number of hours for that representative hour.
+
+ ### Hourly Resource Data
+
+ - **AvailabilityDER.csv**: Availability of DER units (fraction) by customer type, BTM technology, zone, representative day and hour. Column names are index\_h, index\_m, index\_z, index\_d, followed by represenative hours in index\_t  (for e.g., `index_h, index_m, index_z, index_d, h001, h002` etc.).
+
+ - **AvailabilityNewIPP.csv**: Availability of new units (%) by IPP type, technology type, zone, representative day and hour. Column names are index\_p, index\_k\_new, index\_z, index\_d, followed by represenative hours in index\_t  (for e.g., `index_p, index_k_existing, index_z, index_d, h001, h002` etc.).
+
+ - **AvailabilityOldIPP.csc**: Availability of existing units (fraction) by IPP type, technology type, zone, representative day and hour. Column names are index\_p, index\_k\_existing, index\_z, index\_d, followed by represenative hours in index\_t  (for e.g., `index_p, index_k_existing, index_z, index_d, h001, h002` etc.).
+
+
+### Financial Parameters
+
+#### New Resources
+
+ - **Lifetime_new.csv**: Lifetime of new generation technologies in years. Columns are technology names (for e.g., `gas-ct, gas-cc, upv` etc.).
```

Comment:
```suggestion
 - **Lifetime_new.csv**: Lifetime of new generation technologies in years defined over index\_k\_new, i.e., columns are technology names (e.g., `gas-ct, gas-cc, upv` etc.).
```

Reply:
Revise other parameters similarly, that is, explicitly list what indices they are defined over.